modules/bluefish/consumer/bluefish_consumer.cpp

```diff
@@ -277,32 +475,31 @@ struct bluefish_consumer : boost::noncopyable
 						static_cast<int>(frame.audio_data().size()/channel_layout_.num_channels), 
 						static_cast<int>(channel_layout_.num_channels));
 								
-			blue_->system_buffer_write_async(const_cast<uint8_t*>(reserved_frames_.front()->image_data()), 
-											static_cast<unsigned long>(reserved_frames_.front()->image_size()), 
-											nullptr, 
-											BlueImage_HANC_DMABuffer(reserved_frames_.front()->id(), BLUE_DATA_IMAGE));
+			blue_->system_buffer_write(const_cast<uint8_t*>(reserved_frames_.front()->image_data()), 
+											static_cast<unsigned long>(reserved_frames_.front()->image_size()),  
+											BlueImage_HANC_DMABuffer(reserved_frames_.front()->id(), BLUE_DATA_IMAGE), 
```

Comment:
Should `BlueImage_HANC_DMABuffer` be replaced with `BlueImage_HANC_DMABuffer` here as well or is it correct even though it is about video not sound?